cluster/gce/gci/configure-helper.sh

```diff
@@ -1004,6 +1004,9 @@ function start-kube-apiserver {
     params+=" --kubelet-client-certificate=${APISERVER_CLIENT_CERT_PATH}"
     params+=" --kubelet-client-key=${APISERVER_CLIENT_KEY_PATH}"
   fi
+  if [[ -n "${SERVICEACCOUNT_CERT_PATH:-}" ]]; then
```

Comment:
huh, I guess this works because we'll parse the public key out of the certificate...

https://github.com/kubernetes/kubernetes/blob/master/vendor/github.com/dgrijalva/jwt-go/rsa_utils.go#L55

```go
// Parse the key
var parsedKey interface{}
if parsedKey, err = x509.ParsePKIXPublicKey(block.Bytes); err != nil {
	if cert, err := x509.ParseCertificate(block.Bytes); err == nil {
		parsedKey = cert.PublicKey
	} else {
		return nil, err
	}
}
```

didn't see that coming...